src/main/java/org/mockito/ArgumentMatcher.java

```diff
@@ -161,7 +161,7 @@
      * <p>Only matcher implementations that can conceptually match a raw vararg parameter should override this method.
      *
      * @return the type this matcher handles. The default value of {@link Void} means the type is not known.
-     * @since 5.0.0
+     * @since 4.11.0
```

Comment:
Updated, as these were now released in 4.11.0, not 5.0.0

Reply:
FYI, seems you're missing the 4.11.0 tag in the repo.

Reply:
Ah yes good point. With regards to the tag, `v4.11.0` was released: https://github.com/mockito/mockito/releases/tag/v4.11.0